pkg/agent/util/net_linux.go

```diff
@@ -33,16 +33,38 @@ import (
 	"k8s.io/klog/v2"
 )
 
+var (
+	// below are meant to be overridden for testing
+	getNS                  = ns.GetNS
+	netNSDo                = ns.NetNS.Do
+	netNSPath              = ns.NetNS.Path
+	netNSClose             = ns.NetNS.Close
+	getVethPeerIfindex     = ip.GetVethPeerIfindex
+	netlinkByIndex         = netlink.LinkByIndex
```

Comment:
For the mock functions targeting at netlink.XXX, is it possible to leverage pkg/agent/util/netlink.Interface, then no need to add so many mock functions?

Reply:
Utilized `netlink.Interface`, used `expectedFunctionCalls`.